Razor task ran into copy failure.

Please add a retry or SkipUnchangedFiles or fix the race condition.D:\a\1\s\src\RazorSdk\Tasks\Microsoft.NET.Sdk.Razor.Tasks.csproj(107,5): error MSB3026: Could not copy "D:\a\1\s\src\RazorSdk\Sdk\Sdk.targets" to "D:\a\1\s\artifacts\bin\Debug\Sdks\Microsoft.NET.Sdk.Razor\Sdk\Sdk.targets". Beginning retry 1 in 1000ms. The process cannot access the file 'D:\a\1\s\artifacts\bin\Debug\Sdks\Microsoft.NET.Sdk.Razor\Sdk\Sdk.targets' because it is being used by another process.  [D:\a\1\s\src\RazorSdk\Tasks\Microsoft.NET.Sdk.Razor.Tasks.csproj]

https://dev.azure.com/dnceng-public/public/_build/results?buildId=96657&view=results <-- first build failed here, I have the binlog if you need it.

--- a/src/RazorSdk/Tasks/Microsoft.NET.Sdk.Razor.Tasks.csproj
+++ b/src/RazorSdk/Tasks/Microsoft.NET.Sdk.Razor.Tasks.csproj
@@ -101,6 +101,7 @@
   </Target>
 
   <Target Name="CopyAdditionalFilesToLayout"
+          Condition="'$(TargetFramework)' == ''"
           DependsOnTargets="PrepareAdditionalFilesToLayout"
           AfterTargets="Build" Inputs="@(LayoutFile)"
           Outputs="@(LayoutFile->'$(PackageLayoutOutputPath)%(TargetPath)')">

